views/facetsWithArticles/clusteredImages/fiveA.hbs

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html>
+	{{#> head title='Clustered Images - v5a' subtitle='' }}
+	<style type="text/css">
```

Comment:
If CSS can be reused between templates it should be saved as a separate file

Reply:
Good idea, done